src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -717,6 +717,283 @@ For example, if the page that was copied from is called `Leads`, then the newly
+    "y": 150,
+    "w": 1105,
+    "h": 685
+  },
+  "annotations": [{
+    "x": 670,
+    "y": 450,
+    "w": 10,
+    "h": 13
+  }]
+}
+----
+
+A modal will open with two sub-sections: `Upload icon` and `Or add link to icon`.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-icon.png",
+    "alias": "page-edit-configure-icon-view-modal",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 570,
+    "y": 40,
+    "w": 780,
+    "h": 330
+  }
+}
+----
+
+[NOTE]
+====
+The currently supported icon format is `.icon`.
+`.svg` isn't supported yet.
+====
+
+In the `Upload icon` sub-section you can drag and drop or click on the button and choose an icon.
+In the `Or add link to icon` sub-section you can add a link to an icon and preview it.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-icon.png",
+    "alias": "page-edit-configure-icon-preview",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 570,
+    "y": 40,
+    "w": 780,
+    "h": 330
+  },
+  "annotations": [{
+    "x": 1300,
+    "y": 240,
+    "w": 40,
+    "h": 40
+  }]
+}
+----
+
+After uploading an icon a third sub-section will appear.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-uploaded-icon.png",
+    "alias": "page-edit-configure-uploaded-icon-section",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 558,
+    "y": 35,
+    "w": 800,
+    "h": 570
+  },
+  "annotations": [{
+    "x": 580,
+    "y": 300,
+    "w": 760,
+    "h": 180
+  }]
+}
+----
+
+[NOTE]
+====
+After uploading an icon, it will be automatically selected
+====
+
+In it you can select 
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-selected-icon.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 555,
+    "y": 43,
+    "w": 796,
+    "h": 575
+  }
+}
+----
+
+or unselect an icon.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-uploaded-icon.png",
+    "alias": "page-edit-configure-uploaded-icon-unselected",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 555,
+    "y": 43,
+    "w": 796,
+    "h": 575
+  }
+}
+----
+
+If an icon is selected and btn:[Save and close] is clicked, the selected icon will be applied in the `Page icon` section.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-set-icon.png",
+    "alias": "page-edit-configure-set-icon-focus-page-content",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 555,
+    "y": 150,
+    "w": 1100,
+    "h": 685
+  },
+  "annotations": [{
+    "x": 585,
+    "y": 440,
+    "w": 110,
+    "h": 105
+  }]
+}
+----
```

Comment:
I think this is too many details; how to select/unselect, being autoselected should be intuitive enough from the interface.

What I'd do is remove the selected bit of code and update `After uploading an icon a third sub-section will appear.` to something like `After uploading an icon, available icons are shown at the bottom of the modal window.`